CODEOWNERS

```diff
@@ -1,6 +1,23 @@
-*	@joroshiba @noot @SuperFluffy @Fraser999
+/.github/  @astriaorg/infra
+/containerfiles/  @astriaorg/infra
+/charts/  @astriaorg/infra
+/dev/  @astriaorg/infra
+.dockerignore  @astriaorg/infra
 
-/.github/	@astriaorg/infra
-/containerfiles/	@astriaorg/infra
-/charts/ @astriaorg/infra
-/dev/ @astriaorg/infra
+*.rs  @astriaorg/rust-reviewers
+Cargo.toml  @astriaorg/rust-reviewers
+Cargo.lock  @astriaorg/rust-reviewers
+rust-toolchain  @astriaorg/rust-reviewers
+.cargo/  @astriaorg/rust-reviewers
+nextest.toml  @astriaorg/rust-reviewers
+rusfmt.toml  @astriaorg/rust-reviewers
+
+*.proto @astriaorg/api-reviewers
+buf.lock @astriaorg/api-reviewers
+buf.yaml @astriaorg/api-reviewers
+buf.work.yaml @astriaorg/api-reviewers
+
+specs/  @astriaorg/engineering
+justfile  @astriaorg/engineering
+
+* @joroshiba @SuperFluffy @noot
```

Comment:
add @Fraser999 ?

Reply:
This basically covers everything not covered by anything else explicitly. Fraser is a part of the rust reviewers group. I think this is appropriate for now.